index.js

```diff
@@ -3,30 +3,78 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  for (const roll of rolls) {
+    if (typeof roll === 'number') {
+      return typeof roll === 'number'
+    } else if (rolls.length === 0) {
+      return rolls.length === 0
+    } else {
+      return false; //will return false if either above are false
+    }
+  }
+  return true; // will return true if empty array is given and there is nothing to execute in for loop
+}
+
+
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+ for (const roll of rolls) {
+  if (rolls.includes(value) == true) {
```

Comment:
try to avoid using the loosely equals operator `==`. It can cause issues in your code due to misunderstanding. For instance if I changed your conditional here to `5 == true` it would still pass because `5` is a truthy value and therefore loosely equal to `true`. Make sure to use strictly equals `===` or strictly does not equal `!==`